### PR TITLE
docker setup guide for v2

### PR DESCRIPTION
**Pull request by CluEleSsUK**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -2,19 +2,17 @@
...
-
 ## Creation of a keypair
 
 Pull the latest drand image:
 ```shell
-docker pull ghcr.io/drand/go-drand:v2.0.7-testnet
+docker pull ghcr.io/drand/go-drand:v2.0.1
 ```
+> [!NOTE]
+> If you want to run drand locally without TLS, you should use the `ghcr.io/drand/go-drand-local:v2.0.1` image instead!
```
Comment (AnomalRoil) [inline docker/README.md:15]: I think this would warrant a github action step to update this text in our publish image GHA, WDYT?

**Review by AnomalRoil — COMMENTED** [review]
I wonder whether we should be using something like the "latest"  tag for docker images?

Because otherwise we need to update all of these files with each new tagged release, no?

File: docker/README.md (hunk 1 above)
Comment (CluEleSsUK) [reply docker/README.md:15]: yep, good idea!

**Comment by CluEleSsUK** [comment]
I’ve heard mixed views on the latest tag - some claim it’s an antipattern. Personally it makes sense to me but I wonder what @GlacierWalrus has to say on the matter

**Comment by GlacierWalrus** [comment]
> I’ve heard mixed views on the latest tag - some claim it’s an antipattern. Personally it makes sense to me but I wonder what @GlacierWalrus has to say on the matter

Nothing wrong with us keeping a `latest` tag imo. `latest` is only a problem if you don't treat it with due care. I'm sure people have documented comprehensive reasons on their blogs about why you shouldn't use it, my strategy is only to use `latest` for non-serious applications.

**Review by AnomalRoil — APPROVED** [review]

**Pull request merged**